x-pack/plugins/reporting/public/management/components/view_in_app_link.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FunctionComponent } from 'react';
+import { stringify } from 'query-string';
+import { i18n } from '@kbn/i18n';
+import { EuiButtonIcon, EuiToolTip } from '@elastic/eui';
+import React from 'react';
+
+import { getRedirectAppPath } from '../../../common/constants';
+import { buildKibanaPath } from '../../../common/build_kibana_path';
+import { useKibana } from '../../shared_imports';
+import { Job } from '../../lib/job';
+
+interface Props {
+  job: Job;
+}
+
+const i18nTexts = {
+  label: i18n.translate('xpack.reporting.listing.table.viewInAppLinkIconLabel', {
+    defaultMessage: 'Open this report in the Kibana app that requested it.',
```

Comment:
I think this needs some copy review. The idea here is that you we can send users from the reporting management UI back to the app where they requested the report _with_ the state of the app when they requested it.

Reply:
This makes it sound like users can open and view the report in the Kibana App. Is this the intended behavior? If yes, then how about `Open the report in the Kibana App where you generated the report.`

If not, how about `Open the Kibana App where you generated the report.`

Reply:
It's kind of something between those two, in the sense that you're opening the app where you requested it but you aren't literally looking at the generated report. Rather the app as it was when the report was requested.

I think, drawing from your second idea since I think it works a bit better:

`Open the Kibana App where this report was generated.`

I think it would be nice to not scope the copy to the specific user, I can see how in the future you could see reports not generated by you.

Let me know what you think!